plugins/reporters/evaluated-model/src/main/kotlin/EvaluatedModelReporter.kt

```diff
@@ -42,11 +43,12 @@ class EvaluatedModelReporter : Reporter {
 
     override val type = "EvaluatedModel"
 
-    override fun generateReport(input: ReporterInput, outputDir: File, options: Map<String, String>): List<File> {
-        val evaluatedModel = EvaluatedModel.create(input, options[OPTION_DEDUPLICATE_DEPENDENCY_TREE].toBoolean())
+    override fun generateReport(input: ReporterInput, outputDir: File, config: PluginConfiguration): List<File> {
+        val evaluatedModel =
+            EvaluatedModel.create(input, config.options[OPTION_DEDUPLICATE_DEPENDENCY_TREE].toBoolean())
```

Comment:
Nit: I propose to wrap like

        val evaluatedModel = EvaluatedModel.create(
            input,
            config.options[OPTION_DEDUPLICATE_DEPENDENCY_TREE].toBoolean()
        )

for readability.